pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceConsumer.java

```diff
@@ -176,11 +176,11 @@
         @Parameter(names = {"-ioThreads", "--num-io-threads"}, description = "Set the number of threads to be " +
                 "used for handling connections to brokers, default is 1 thread")
         public int ioThreads = 1;
-    
+
         @Parameter(names = {"--batch-index-ack" }, description = "Enable or disable the batch index acknowledgment")
         public boolean batchIndexAck = false;
 
-        @Parameter(names = { "-pm", "--pool-messages" }, description = "Use the pooled message")
+        @Parameter(names = { "-pm", "--pool-messages" }, description = "Use the pooled message", arity = 1)
```

Comment:
Good catch！Should other option that type is boolean be handled like this? such as --replicated, --auto_ack_chunk_q_full, --batch-index-ack etc.

Reply:
It would be a breaking change. This is only necessary for boolean options that default to true.